src/events/components/SearchModule/DateRangeInput.tsx

```diff
@@ -0,0 +1,65 @@
+import { DateTime } from 'luxon';
+import React, { FC, MutableRefObject, useRef } from 'react';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
```

Comment:
I hope there is a better way to add this CSS, since I had to add CSS-importing to webpack-config for this to work

Reply:
Are you maybe able to import that css into a css file with
`@import "/node-modules/react-datepicker/lib/react-datepicker.css"`and load it?

Reply:
The problem is that webpack wasn't able to load CSS files with import, until I added it with this PR, if I import it through a `.less`-file (which I think you are meaning) it would add the hashes to the CSS-classes, and thus the CSS would have no effect

Reply:
This is an _ok_ way to import external CSS, I guess.
Doing to through Less is cumbersome because it is configured with CSS Modules.

Just make sure it works with SSR.

Reply:
Update: does not appear to be working with SSR, getting multiple errors, from rejected promises trying to fetch CSS, but the main page loads without it, `localhost:8080/events` gets a rejected promise, document is not defined when trying to check if the browser is old IE, etc.

Reply:
Is it possible to import it into Less, with `:global`?

Reply:
Not really. `:global` has to be applied to each class. There are postcss plugins for doing that for you, but I think that becomes too much of a convoluted solution.